pkg/vsphere/vm/vm.go

```diff
@@ -611,6 +614,23 @@ func (vm *VirtualMachine) DisableDestroy(ctx context.Context) {
 
 	err := m.DisableMethods(ctx, obj, method, "VIC")
 	if err != nil {
-		log.Warnf("failed to disable method %s for %s: %s", method[0].Method, obj[0], err)
+		log.Warnf("Failed to disable method %s for %s: %s", method[0].Method, obj[0], err)
+	}
+}
+
+// EnableDestroy attempts to enable the VirtualMachine.Destroy_Task method
+// so that the VC UI can enable the VM "delete" action.
+func (vm *VirtualMachine) EnableDestroy(ctx context.Context) {
```

Comment:
Minor suggestion: it would be helpful to have this func return an error so the caller can decide what to do in that case. This may not be super relevant for this change, but it would be for extensibility.